#false = frei
bool front 
bool left
bool right
int8 close #0 = nix 1=links 2=mitte 3=rechts 
